### PR TITLE
fixed search bar mobile [ issue 973 ]

### PR DESCRIPTION
**Pull request by vaibhav-xt**
## Related Issue
    Issue No. - #973 

## Description
    The search bar width on mobile was not good. 

## Screenshots
    
Screenshot (482)

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2035,11 +2035,17 @@
     },
     "Projects": [
       {
-        "link": "https://github.com/prasanta10/Blog_wesbsite",
+        "link": "https://github.com/Utkers-h/Coffe_and_Wifi",
+        "title": "Coffe and Wifi",
+        "description": "This project will help developers to find suitable cafes with proper power supply , that will help them to work efficiently while enjoying the best drinks, and atmosphere that cities and towns can offer.",
+        "tech": ["Html", "CSS", "Flask-WTF", "Flask-Bootstrap"]
+      },
```
Comment (priyankarpal) [inline src/DB/projects.json:2042]: why did you add the code here?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (vaibhav-xt) [reply src/DB/projects.json:2042]: I think while resolving conflict this happened.

Comment (priyankarpal) [reply src/DB/projects.json:2042]: ok then remove extra code

**Comment by vaibhav-xt** [comment]
> close this PR & make sure to update your branch then create PR 

But please this time mege it immediately

**Comment by priyankarpal** [comment]
> > close this PR & make sure to update your branch then create PR
> 
> But please this time merge it immediately

how can I merge this PR? there are some errors.

**Comment by vaibhav-xt** [comment]
> > > close this PR & make sure to update your branch then create PR
> > 
> > 
> > But please this time merge it immediately
> 
> how can I merge this PR? there are some errors.
I was saying that when I make another pull request then

**Comment by priyankarpal** [comment]
so we added new features & removed the suggestion & search bar also got fixed. I think we don't need to change it right now. 

image
